src/main.tsx

```diff
@@ -2,10 +2,20 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./globals.css";
 import App from "./App";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import StartPage from "./pages/Start/Start";
+import NameInput from "./pages/NameInput/NameInput";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<StartPage />} />
```

Comment:
Is  the path attribute missing here?

Reply:
Without the index the outlet area is blank. 

Here's the explanation from the original [documentation](https://reactrouter.com/docs/en/v6/getting-started/tutorial):
_Notice it has the index prop instead of a path. That's because the index route shares the path of the parent. That's the whole point--it doesn't have a path.

Maybe you're still scratching your head. There are a few ways we try to answer the question "what is an index route?". Hopefully one of these sticks for you:

Index routes render in the parent routes outlet at the parent route's path.
Index routes match when a parent route matches but none of the other children match.
Index routes are the default child route for a parent route.
Index routes render when the user hasn't clicked one of the items in a navigation list yet._